lib/galaxy/webapps/galaxy/api/sanitize_allow.py

```diff
@@ -1,31 +1,34 @@
 """
 API operations allowing clients to retrieve and modify the HTML sanitization allow list.
 """
+
 import logging
 from typing import (
     Any,
     Dict,
 )
 
-from galaxy import web
-from galaxy.webapps.base.controller import BaseAPIController
+from galaxy.managers.context import ProvidesUserContext
+from galaxy.webapps.galaxy.api import DependsOnTrans
+from . import Router
 
 log = logging.getLogger(__name__)
 
+router = Router(tags=["sanitize_allow"])
```

Comment:
```suggestion
router = Router(tags=["configuration"])
```

Reply:
@mvdbeek not sure why I should do this? shouldn't the router tags match the endpoint?

Reply:
there's no need for this to be a top level entry in the documentation (that's what tags are for), you're just changing configuration values here.